preview-env/clean/action.yml

```
@@ -58,6 +58,9 @@ inputs:
 
       ⏯️ **Want to keep preview environment(s) for longer ?**
       Simply push a new commit or restart the deployment [workflow]({checks-url}) to **postpone the shutdown to `{ttl-days} day(s)` from now**.
+
+      🔁 **Want to keep preview environment(s) permanently ?**
+      Consider deploying a static environment associated with a long-running branch.
```

what exactly do you mean by it?
It sounds like we have "stable" preview environments but we don't have that.

---

Indeed it is not very clear. I was actually thinking of stable branches when I wrote long-running branches. The idea was to anticipate the needs of teams to deploy environments for their stable branches and point them towards the right solution and avoid them hijacking the use of dynamic preview environments as [Operate](https://github.com/camunda/operate/pull/4741) has done. Operate is a special case because preview environments are already activated for stable branches (this PR needs to be closed without triggering a teardown).

I could change to:
```suggestion
      🔁 **Want to keep preview environment permanently for a stable branch?**
      Consider deploying a static environment and reach Infra team for support.
```

Or I can just add as a reminder in the announcement. 

What do you think ?

---

Your suggestion is more clear 👍.
Yeah always wondered what that Operate PR was about since `stable/**` is already supported at them.

---

but I'm still not sure whether to keep it.
E.g. Web-Modeler asked recently about having a branch deployed for a longer period, which they I guess would consider stable in that case but I don't think we should create temporary exceptions for those "stable" branches.
`stable/**` I see as an exclusion since we already have those as exceptions in Operate and Tasklist and the images are tagged accordingly.

the `stable/**` should already work for them out of the box, so at them it's simply a misuse of the preview-environments.

---

so overall, would remove it to not create a potential of us doing exceptions all the time for feature branches that need to "live" longer.

Or change it to something like `Want to keep a preview environment for a release branch?`

---

I see 👍 . Let's delete it for now. We'll adjust it later if there is a need.